### PR TITLE
Add 497: HTTP Request sent to HTTPS port

### PR DESCRIPTION
**Pull request by zurgeg**

### CODE REVIEW

**Comment by rogeriopvl** [comment]
Hi @zurgeg , thanks for the contribution. There are a couple of issues with the PR that we need to clear out before merging:

- The image is heavy (almost 1Mb, if you check most of our images are around 40Kb), please run an optimizer before adding it
- You used the portrait template, but the image seems more suited for the landscape template (please check the template instructions on the README file)
- You need to add the status code in the `src/lib/statuses.js` file, otherwise it won't show up in the website

Cheers!

**Comment by zurgeg** [comment]
> 
> 
> Hi @zurgeg , thanks for the contribution. There are a couple of issues with the PR that we need to clear out before merging:
> 
>     * The image is heavy (almost 1Mb, if you check most of our images are around 40Kb), please run an optimizer before adding it
> 
>     * You used the portrait template, but the image seems more suited for the landscape template (please check the template instructions on the README file)
> 
>     * You need to add the status code in the `src/lib/statuses.js` file, otherwise it won't show up in the website
> 
> 
> Cheers!

Thanks for the advice! I didn't really read the README much (sorry for that 😅), but I actually used [esmBot](https://github.com/esmBot/esmBot) to create the image, I will recreate this taking that in mind!

**Comment by zurgeg** [comment]
@rogeriopvl I fixed the issues you told me about, how's it looks now?

File: src/lib/statuses.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ const statuses = {
   444: { code: 444, message: 'No Response' },
   450: { code: 450, message: 'Blocked by Windows Parental Controls' },
   451: { code: 451, message: 'Unavailable For Legal Reasons' },
+  497: { code: 497, message: 'HTTP Request sent to HTTP port' },
```
[the remark this answers is not in the document]
Comment (zurgeg) [reply src/lib/statuses.js:51]: Whoops haha

**Comment by zurgeg** [comment]
@rogeriopvl I fixed the issue you mentioned, how's it now?

File: src/lib/statuses.js [hunk 2]
```diff
@@ -48,6 +48,7 @@ const statuses = {
   444: { code: 444, message: 'No Response' },
   450: { code: 450, message: 'Blocked by Windows Parental Controls' },
   451: { code: 451, message: 'Unavailable For Legal Reasons' },
+  497: { code: 497, message: 'HTTP Request sent to HTTPS port' },
```
Comment (rogeriopvl) [inline src/lib/statuses.js:51]: Sorry, should have seen this before. Can you also update the capitalization in the image text? Should be ready to merge right after.

```suggestion
  497: { code: 497, message: 'HTTP Request Sent to HTTPS port' },
```

Comment (zurgeg) [reply src/lib/statuses.js:51]: Alright, just give me one minute :)

**Comment by zurgeg** [comment]
@rogeriopvl Alright, should be everything right?

**Comment by rogeriopvl** [comment]
😅 the image has the word "port" in lowercase

**Comment by zurgeg** [comment]
> 
> 
> 😅 the image has the word "port" in lowercase

Loool, oops

**Comment by zurgeg** [comment]
Alright, @rogeriopvl I fixed that

**Comment by rogeriopvl** [comment]
Great @zurgeg , I'm going to merge this! Thanks! 👍

**Comment by rogeriopvl** [comment]
@all-contributors add @zurgeg for code

**Pull request merged**